content/participants/ttjc@jeringeorge98/index.md

```diff
@@ -0,0 +1,91 @@
+
+I also want to know more about Open Source development and would like to conntribute in future as most of the things i have learnt has been with the help of Open source software.
+
+## something interesting about you?
+If I am not coding ,Iam reading books on economics and finance,reading about dispruptive technologies like blockhain,love playing chess and discussing football.
+
+### help I need?
+I am looking to explore alternative to REST APIS like GraphQL,so guidance in that would be appreciated.
+
+
+
+### how to contact me?
+
+- Catch me on [Twitter](https://twitter.com/LastJERI1)
+- Catch me on [LinkedIn](https://www.linkedin.com/in/jeringeorge98/)
+
+### Projects
+
+Few Projects I have built. Will be updating projects as and when I am done.
+
+My projects:
+
+#### A Social Networking Website 
+
+_Description_:  Often People are confused as to which project would help them cover most frontend and backend skills together in one project.I found a simple social networking web application helped me cover most frontend and backend skills and was also a lot of fun in building this application.This application allows users to make an account,signin,follow/unfollow other users,view and comment on their posts.
+
+_Stack_: MERN stack.Frontend was developed with the help of REACTJS framework.Backend was developed with ExpressJs and NodeJs.MongoDb was used as the database Used the mongodb atlas which is the mongodb cloud service.
+
+_Hosted link_ : Not hosted currently
+
+_Github link_:https://github.com/jeringeorge98/Fullstack_social_network_website
+
+#### Bug tracking Software 
+
+_Description_:
+
+A web  application aimed at keeping a record of Bugs raise by developers ,Developers can raise issues ,assign it to other developers,Issues can be closed once its fixed.Bulding this application was mainly focused around the fact of getting familiar with pure Javascript before getting into a particular framework.
+
+_Stack_:
+HTML,CSS and vanillaJs ,mainly a frontend application.Data is stored in the browsers local storage.
+
+_Hosted link_ : Not hosted currently
+
+_Github link_:https://github.com/jeringeorge98/Bug-tracking-software-with-vanilla-Js-
+
+#### Simple React-Native application and Authentication with firebase  
+
+_Description_:
+Mobile application built with React Native ,which demonstrates simple authentication ,Users are authetcated with firebase api.
+_Stack_:
+React-Native ,Firebase
+
+_Github link_:https://github.com/jeringeorge98/Bug-tracking-software-with-vanilla-Js-
+
+### Internship
+Worked at a startup for 7 months were I worked on a co-working mobile application.The application was built with REACT NATIVE ,I was involved throughout the process from production to development.
+
+_Roles ad Responsibilties_: Included buidling frontend User interfaces on react native ,leveraging 3 party APIS, and building RESTful APIS to connect to the backend.
```

Comment:
Some more typos:
* and
* building

Reply:
Sorry for the typos i will correct them and send it again

Reply:
Hey khusharth i have resolved the typos and changed the link please take alook and accept my request.

Thank You!